curvefs/src/client/metric/client_metric.h

```diff
@@ -109,6 +109,34 @@ struct FSMetric {
           userRead(prefix, fsName + "_userRead") {}
 };
 
+struct S3Metric {
+    const std::string prefix = "curvefs_s3";
```

Comment:
there no need to define `prefix` as data member

Reply:
In order to unify the style, there is only one instance of a metric at present, so there will be no waste of space